packages/react-components/src/atoms/MultiSelect.tsx

```diff
@@ -159,6 +164,32 @@ const MultiSelect = <T extends MultiSelectOptionsType>({
     inputRef?.getWrappedInstance?.().blur?.();
   };
 
+  const { validationMessage, validationTargetProps, validate } =
+    useValidation<HTMLInputElement>(
+      customValidationMessage,
+      getValidationMessage,
+    );
+
+  const initialRender = useRef(true);
+  const checkValidation = useCallback(() => {
+    if (initialRender.current) {
+      initialRender.current = false;
+    } else {
+      validate();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Why isn't initialRender in this list? Shouldn't it be called inputRef?

Reply:
Following the approach we have in Dropdown.tsx, we don't want to validate on the first render so we don't display the error message directly.